free-2.md

```diff
@@ -8,44 +8,103 @@ good way to meet their needs.
 them?_
 
+Playlists of songs are currently a simple ordering of specific tracks to play.
+If a music listener wants to hear new music without explicitly saying what
+they want to hear they may turn to discovery services like Spotify or Pandora;
+however, these services lack fine-grained control over what songs you listen to
+usually relying on factors like what you've listened to in the past and whether
+or not you thumbs up a song.
+
+It would be extrmeley nice if one could specify with more paramaeters what they
+want to listen to. For instance, specify a genre and a deviation from that genre,
+the number of songs per artist or band to be played in given amount of time, or
+beats per minute of a song in the playlist.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+A simple thumbs up or thumbs down (or even an invisble predictive algorithm) do
+not offer enough control. With a DSL, one could specify what they want to hear
+with more expressiveness and clarity. This DSL would need to be more natural
+feeling than most programming langugaes to engage the most number of users.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+This was a personal idea generated out of my frustration with Spotify's auto playlists
+being too redundent and wanting a way to specify weights for artists and bands
+that I wanted to listen to.
+
+I ran it by a few friends to get their feedback and they thought it would be a
+neat idea to pursue and implement.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Intelligent and controllable playlist creation.
 
 ### Interface (syntax)
-_How might the user interact with the language? What does programming look 
-like? Why is this the right way to interact with the problem domain?_ 
-
+_How might the user interact with the language? What does programming look
+like? Why is this the right way to interact with the problem domain?_
+
+Since this langugae would target people with a variety of backgrounds in programming&mdash;
+mostly people without experience&mdash;I imagine a Scratch-like visual language.
+
+The blocks however, would be more tailored to music playlists. Some examples include:
++ artist: specifies an artist of a song
++ band: specifies a band
++ song: specifies a song name directly
++ not: negates an expression so you could express things along the lines of
+       "I don't want to listen to ABBA".
++ redundency: number of songs belonging to an entity and a time in which you
+  would like to hear them
++ divergence: specify the divergence factor from the current song
++ bpm: specifies beats per minute of the song
+
+While I think it may be easier to implement a DSL that looks like a programming
+langugae, a visual language would be easier to learn and an artistic
+component to the playlist. Each playlist would, through its visual prgamming,
+have its own artwork/fingerprint.
 
```

Comment:
I'm not sure how necessary it is to specify tempo. I rarely find myself thinking I want something at 142 bpm. Maybe just specify "fast" or "slow", or tempo markings like "allegro" if the person knows music (or Italian).